docs/narr/urldispatch.rst

```diff
@@ -165,8 +165,8 @@ The above pattern will match these URLs, generating the following matchdicts:
 
 .. code-block:: text
 
-    foo/1/2        -> {'baz':u'1', 'bar':u'2'}
-    foo/abc/def    -> {'baz':u'abc', 'bar':u'def'}
+    foo/1/2        -> {'baz':'1', 'bar':'2'}
```

Comment:
FYI, when doing search/replace, the actual output of all these examples should have a space between the colon and the value. This is the first time I caught this in this PR, so I went back and checked other places where you removed "u" from strings. I've pushed a commit to address these and those examples.

```
>>> foo = {'baz':'1', 'bar':'2'}
>>> foo
{'baz': '1', 'bar': '2'}
```